src/placement/tests/place_obj_common.c

```diff
@@ -772,6 +772,10 @@ get_object_classes(daos_oclass_id_t **oclass_id_pp)
 		return -1;
 
 	length = daos_oclass_names_list(str_size, oclass_names);
+	if (length != -1) {
```

Comment:
should probably be length < 0.

Reply:
Yeah, the value is unsigned and I didn't want to pull in other changes